base_layer/transactions/src/consensus.rs

```diff
@@ -25,11 +25,16 @@ use chrono::{DateTime, Duration, Utc};
 use std::ops::Add;
 
 // This is the our target time in seconds between blocks
-pub const TARGET_BLOCK_INTERVAL: u64 = 60;
+pub const TARGET_BLOCK_INTERVAL: u64 = 120;
 // When doing difficulty adjustments and FTL calculations this is the amount of blocks we look at
-pub const DIFFICULTY_BLOCK_WINDOW: u64 = 150;
+pub const DIFFICULTY_BLOCK_WINDOW: u64 = 120;
```

Comment:
```suggestion
pub const DIFFICULTY_BLOCK_WINDOW: u64 = 90;
```
For t = 120, N should be 90
[https://github.com/zawy12/difficulty-algorithms/issues/3](https://github.com/zawy12/difficulty-algorithms/issues/3)

Reply:
Change applied.